packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -102,10 +102,10 @@ export const SYNC_TOLERANCE_EPOCHS = 1;
  * Post this time, race execution and builder to pick whatever resolves first
  *
  * Empirically the builder block resolves in ~1.5+ seconds, and execution should resolve <1 sec.
- * So lowering the cutoff to 2 sec from 3 seconds to publish faster for successful proposal
- * as proposals post 4 seconds into the slot seems to be not being included
+ * So lowering the cutoff to 2.5 sec from 3 seconds to publish faster for successful proposal
+ * as proposals post 4 seconds into the slot will likely be orphaned due to proposer boost reorg.
  */
-const BLOCK_PRODUCTION_RACE_CUTOFF_MS = 2_000;
+const BLOCK_PRODUCTION_RACE_CUTOFF_MS = 2_500;
```

Comment:
In my opinion increases this by 500ms should be fine because the previous value was absolute, meaning no matter at which time we started the block production race it would always give 2 seconds while now it's most of the time just 1.5-1.7 seconds since the cutoff it based on seconds into slot.

We also have a strict timeout on builder header request now which means this cutoff should rarely be relevant.

In addition, this value was chosen when we used JSON to pass the block around vc<>bn which added at least 500ms more to publish flow compared to using SSZ + we no longer flood publish blocks and blobs which makes it even less likely to orphan blocks.

Reply:
> pass the block around vc<>bn which added at least 500ms more to publish flow compared to using SSZ 

ok this makes sense to me. so lgtm